runtime/compilers/rillv1/parse_api.go

```diff
@@ -77,6 +81,26 @@ func (p *Parser) parseAPI(node *Node) error {
 	}
 	node.Refs = append(node.Refs, resolverRefs...)
 
+	securityRules, err = tmp.Security.Proto()
+	if err != nil {
+		return fmt.Errorf("failed to parse security rules: %w", err)
+	}
+	for _, rule := range securityRules {
+		if rule.GetAccess() == nil {
+			return fmt.Errorf("the 'api' resource type only supports 'access' security rules")
+		}
+	}
+
+	if tmp.SkipResolverSecurity {
+		// Skip security rules for nested resolvers
+		propsMap := resolverProps.AsMap()
+		propsMap["skip_nested_security"] = tmp.SkipResolverSecurity
+		resolverProps, err = structpb.NewStruct(propsMap)
+		if err != nil {
+			return fmt.Errorf("failed to update resolver properties: %w", err)
+		}
+	}
```

Comment:
In the earlier comment:
> Can you make this a typed field on the APISpec instead? Since it's a property that should be handled upstream, not inside the individual resolver implementations.

I meant that instead of adding it in the resolver props, better to put it directly on the `APISpec` in `proto/rill/runtime/v1/resources.proto` and then set it like this around line 112:
```
r.APISpec.SkipNestedSecurity = tmp.SkipNestedSecurity
```